### PR TITLE
Checkbox incomplete

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
## Description

An better example for incomplete checkbox.

-----

-----

Capture d’écran 2023-11-02 à 11 06 54

### CODE REVIEW

File: stories/documentation/forms/checkbox-legacy/checkbox-incomplete.stories.ts [hunk 1]
```diff
@@ -8,10 +8,26 @@ export default {
 } as Meta;
 
 function getTemplate(args: CheckboxIncompleteStory): string {
-	return `<label class="checkbox">
-	<input class="checkbox-input is-incomplete" type="checkbox" name="checkboxList1" checked>
-	<span class="checkbox-label">Checkbox</span>
-</label>`;
+	return `
+		<label class="checkbox">
+			<input class="checkbox-input incomplete" type="checkbox" checked="checked" aria-checked="mixed" />
```
Comment (AlexiaAlarza) [inline stories/documentation/forms/checkbox-legacy/checkbox-incomplete.stories.ts:13]: the class is "is-incomplete" ?

File: stories/documentation/forms/checkbox-legacy/checkbox-incomplete.stories.ts [hunk 2]
```diff
@@ -8,10 +8,26 @@ export default {
...
+		<ul class="u-listReset u-marginLeftM">
+			<li>
+				<label class="checkbox">
+					<input class="checkbox-input" type="checkbox" checked="checked" />
+					<span class="checkbox-label">Checkbox</span>
+				</label>
+			</li>
+			<li>
+				<label class="checkbox">
+					<input class="checkbox-input is-incomplete" type="checkbox" />
```
Comment (AlexiaAlarza) [inline stories/documentation/forms/checkbox-legacy/checkbox-incomplete.stories.ts:25]: do you need the "is-incomplete" class here ?

Comment (vvalentin-lucca) [reply stories/documentation/forms/checkbox-legacy/checkbox-incomplete.stories.ts:25]: No! Thx.

File: stories/documentation/forms/checkbox-legacy/checkbox-incomplete.stories.ts (hunk 1 above)
Comment (vvalentin-lucca) [reply stories/documentation/forms/checkbox-legacy/checkbox-incomplete.stories.ts:13]: Both work.

**Review by AlexiaAlarza — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2396/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/8348/ui-diff)

**Review by ymalie-lucca — APPROVED** [review]

**Pull request merged**